src/components/DragCaptureZone/DragCaptureZone.jsx

```diff
@@ -45,13 +45,20 @@ const DragCaptureZone = createClass({
 		 * Signature: `({ dx, dy, pageX, pageY }, { event, props }) => {}`
 		 */
 		onDragStart: func,
+		/**
+		 * Called when the drag event is canceled dur to user interaction
```

Comment:
dur -> due
also you could add something about the cancel being triggered by them moving their point of touch off of the screen.